test_runner/src/main/kotlin/ftl/domain/DescribeAndroidTestEnvironment.kt

```diff
@@ -21,7 +22,7 @@ fun DescribeAndroidTestEnvironment.invoke() {
     logLn(AndroidCatalog.devicesCatalogAsTable(projectId))
     logLn(AndroidCatalog.supportedVersionsAsTable(projectId))
     logLn(AndroidCatalog.localesAsTable(projectId))
-    logLn(providedSoftwareAsTable())
+    logLn(fetchSoftwareCatalog().asTable())
```

Comment:
Nothing important but other methods converting data to the table was named ```.toCliTable()``` maybe we can keep one naming conversion? WDYT?

Reply:
@adamfilipow92 sure.